_releases/2022-01-18-1.3.2-released.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Crystal 1.3.2 is released!
+version: 1.3.2
+summary:
+thumbnail: +
+author:
```

Comment:
We've been missing the author(s) in the release notes from 1.3.X. I think it looks nicer with them. We can fix the previous two, to have the two of us in the 1.3.0 and you on the 1.3.1.

```suggestion
author: straight-shoota
```